CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 #### Data migrations
 - [#1009](https://github.com/LayerManager/layman/issues/1009) Fill column `description` in `publications` table in prime DB schema. Value is taken from GeoServer for layers and from filesystem for maps.
 ### Changes
+- [#161](https://github.com/LayerManager/layman/issues/161) [DELETE User](doc/rest.md#delete-user) During user deletion, their publications in public workspace are also removed if the user is the only one with both read and write permissions. For publications where another user has write access, the owner's permissions are removed. If any user has read access, the deletion is aborted.
```

Comment:
Please move details to documentation (/doc/rest.md) and leave only info about new endpoint and method here in changelog.